### DISCUSSION TITLE
Backup restore timeout

### DISCUSSION THREAD

**Discussion by faperreault** [stefanzweifel/laravel-backup-restore#53] [Q&A]
My backup is pretty huge and I have get a `Illuminate\Process\Exceptions\ProcessTimedOutException` since the Process used to restore doesn't seem to have a set timeout other than the default one. (after only 60 seconds)

Is it a known limitation or is there a way around that?

**Comment by stefanzweifel** [comment]
Closing as duplicate from https://github.com/stefanzweifel/laravel-backup-restore/issues/54
